code/drasil-example/Drasil/GamePhysics/README.md

```diff
@@ -32,9 +32,13 @@ README.md
   
 References.hs
   - References for the Game Physics example
-  
+
+Requirements.hs
+  - Requirements for the Game Physics example
+  - currently a work in progress (improvements/changes are being made to the NFRs)
```

Comment:
Great to see Requirements.hs added to the repo.  Saying it is a work in progress is also a good idea.  However, I suggest removing the parenthetical part about NFRs.  That is a very current observation.  It won't age well.  Using the pull request and commit messages the fact that the NFRs have been updated will be captured in the project's history.

Reply:
The date of the update of the README file should also be changed to reflect your edits.